_view/attachChildViews.md

```diff
@@ -3,4 +3,4 @@ header: attachChildViews
 example: View.attachChildViews()
 ---
 
-Removes all child views, then looks at all of the child views that are expected and attaches each view.
+Removes all child views, then looks at all of the expected child views and attaches each view.
```

Comment:
Correct me if I'm wrong...  `attachChildViews` actually looks at the view's HTML generated by the server, finds the child views in there, and then sets them on the view as `childViews`.  Then it attaches each one of those child views.  Yes?